packages/amplify_core/lib/src/category/amplify_storage_category.dart

```diff
@@ -235,3 +235,38 @@ class StorageCategory extends AmplifyCategory<StoragePluginInterface> {
     );
   }
 }
+
+/// {@template amplify_core.amplify_storage_category.subpath_strategy}
+/// Configurable options for `Amplify.Storage.list`.
+/// {@endtemplate}
+class SubpathStrategy with AWSSerializable<Map<String, Object?>> {
```

Comment:
this class can be moved to its own file under`packages/amplify_core/lib/src/types/storage`

Reply:
great callout, updated